dev-python/QtPy/QtPy-1.9.0-r4.ebuild

```diff
@@ -0,0 +1,117 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..8} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Abstraction layer on top of PyQt5 and PySide2 and additional custom QWidgets"
+HOMEPAGE="https://github.com/spyder-ide/qtpy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="
+	pyqt5 pyside2 3d bluetooth dbus charts concurrent datavis declarative
+	designer gui help location multimedia network networkauth opengl positioning
+	printsupport qml quick script scripttools scxml sensors serialport speech
+	ssl sql svg testlib webchannel webengine webkit websockets widgets x11extras
+	xml xmlpatterns
+"
+
+# We have to use at least one of PyQt5 or PySide, but if we want to use bindings
+# that are only available in one of the implementations we have to use that
+# implementation.
+REQUIRED_USE="
+	|| ( pyqt5 pyside2 )
+	pyqt5? (
+		qml? ( declarative )
+		quick? ( declarative )
+	)
+	pyside2? ( declarative? ( qml quick ) )
+
+	bluetooth? ( pyqt5 )
+	dbus? ( pyqt5 )
+	networkauth? ( pyqt5 )
+	ssl? ( pyqt5 )
```

Comment:
This looks weird. PySide2 doesn't support SSL?

Reply:
Fair point, this is indeed wrong